src/scripts.js

```diff
@@ -22,15 +23,9 @@ homeButton.addEventListener('click', cardButtonConditionals);
 favButton.addEventListener('click', viewFavorites);
 cardArea.addEventListener('click', cardButtonConditionals);
 
-
-
 window.onload = onStartup();
 
 function onStartup() {
-  // getData('http://localhost:3001/api/v1/users')
-  getData("http://localhost:3001/api/v1/ingredients")
-  getData("http://localhost:3001/api/v1/recipes")
-
```

Comment:
This is what I was talking about re: the GET calls.

Reply:
I think we don't need them here.